doc/kube-setup-data-ingestion-job.md

```diff
@@ -0,0 +1,16 @@
+The data-ingestion-job is specific to DataSTAGE.
+
+To prep: 
+Fill out the config file with creds here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/data_ingestion_job_config.json`
```

Comment:
what is the format for this file? I think we need an example here to know what fields are required

Reply:
oh, it seems like this might be generated automatically? There should be an instruction before this one to run the command that generates this file if that's the case

Reply:
it would be nice to have a description for each of the config vars too. Also for `github_personal_access_token` it would be nice to list what GitHub permission that token has to have. I don't want to give it admin